docs/CHANGELOG.rst

```diff
@@ -40,7 +40,7 @@ Changed
   ``normalization_type`` input
 - Bump MultiQC to version 1.7.0 in ``multiqc`` process
 - Use ``resolwebio/common:1.1.0`` base image for the
-  ``resolwebio/rnaseq:4.3.0`` Docker image
+  ``resolwebio/rnaseq:4.4.0`` Docker image
```

Comment:
Add this under unreleased section:

Add shRNAde (v1.0) R package to the ``resolwebio/rnaseq:4.4.0`` Docker image